shell/platform/windows/client_wrapper/include/flutter/flutter_window.h

```diff
@@ -0,0 +1,92 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_CLIENT_WRAPPER_INCLUDE_FLUTTER_FLUTTER_WINDOW_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_CLIENT_WRAPPER_INCLUDE_FLUTTER_FLUTTER_WINDOW_H_
+
+#include <string>
+#include <vector>
+
+#include <flutter_windows.h>
+
+#include "plugin_registrar.h"
+
+namespace flutter {
+
+// A data type for window position and size.
+struct WindowFrame {
+  int left;
+  int top;
+  int width;
+  int height;
+};
+
+// A window displaying Flutter content.
+class FlutterWindow {
```

Comment:
I intentionally didn't change the API as part of the first change a) to make it easier to switch the testbed back and forth for testing parity and b) because the new API needs to consider both CoreWindow and HWND as a currency as well as hosting vs full window.  Given how big the change is already, hope it's OK to leave the API as is (with cleanup) and tackle the new API as part of a separate PR?

Reply:
Yes, I figured out post-review when trying it locally that the reason there are stubs to match the old API exactly was probably to make it easier to swap in for testing. That's fine for this version.